editor/src/components/canvas/canvas-strategies/strategies/absolute-duplicate-strategy.spec.browser2.tsx

```diff
@@ -55,12 +63,12 @@ describe('Absolute Duplicate Strategy', () => {
         <div style={{ width: '100%', height: '100%', position: 'relative' }} data-uid='aaa'>
           <div
             style={{ backgroundColor: '#aaaaaa33', position: 'absolute', left: 40, top: 50, width: 200, height: 120 }}
-            data-uid='bbb'
+            data-uid='hello'
             data-testid='bbb'
-          />
-          <div
+            />
+            <div
             style={{ backgroundColor: '#aaaaaa33', position: 'absolute', left: 80, top: 25, width: 200, height: 120 }}
-            data-uid='hello'
+            data-uid='bbb'
             data-testid='bbb'
           />
         </div>
```

Comment:
I think this test change reveals a problem that should be fixed:
if I drag to duplicate `bbb`, I'd expect the duplicated element to appear _below_ the original element in the code.

Reply:
I thought about this, because I was expecting the same thing, but thinking about it it doesn't _actually_ matter which one comes first (the original or the duplicate), as long as the **moved** element comes second (because that gives the impression that the duplicate element has come second). If the only thing showing this to be the original element is the UID then that doesn't matter since that's internal anyway (and will change on the next parse).

The only part where this might become a problem is if it introduces some edge case that can cause selection to switch elements at some point after the duplication (due to the UIDs switching)

Reply:
Fixed in https://github.com/concrete-utopia/utopia/pull/3359/commits/c7b7dd1730fe94b548303f68d595f0747b989934